.github/workflows/docker-hook-api.yml

```diff
@@ -17,6 +17,19 @@ jobs:
 
       - name: Check Out Repo
         uses: actions/checkout@v3
+      
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
I don't seem to have repo access to verify these secrets are loaded in. Do you?

Reply:
Yeah sorry, this was discussed in Slack, they are organization level so we are all set. :)